### PR TITLE
Release 2.3.0

### PR DESCRIPTION
**Pull request by N-Molham**
# Summary

The branch for the v2.2.1 release.

### Includes

* #1683
* #1752
* #1756 (updates to build & release)
* #1745
* #1768
* #1719 
* #1749
* #1764
* #1766
* #1705

### Not included

* #1721 - still a WIP

### Stories

- [x] ~~[CH 55748](https://app.clubhouse.io/skyverge/story/55748) PR #1683~~
- [ ] [CH 67779](https://app.clubhouse.io/skyverge/story/67779/) PR #1721

### Final story

- [ ] [CH 67779](https://app.clubhouse.io/skyverge/story/67779/) PR #1721

## Deploy

- Ensure you have no untracked files or uncommitted changes on your working directory
- Use `npx sake deploy --deployAssets=0` to skip the `shell:svn_commit_assets` action

## After deploy

If you deploy from this branch, before merging the changes into master, follow these steps to fix the tag that `saké` creates:

- `git fetch` to get the latest tags
- `git tag -d {version}` to delete the tag that `sake` created
- `git tag {version} {hash of Facebook for WooCommerce versioning commit}`
- `git push --tags -f` to overwrite the tag on GitHub with the modified tag

### CODE REVIEW

File: assets/js/admin/facebook-for-woocommerce-products-admin.js [hunk 1]
```diff
@@ -370,6 +369,151 @@ jQuery( document ).ready( function( $ ) {
 		}
 
 
+		/**
+		 * Store the original value of the given element for later use.
+		 *
+		 * @since 2.1.5-dev.1
```
Comment (jconroy) [inline assets/js/admin/facebook-for-woocommerce-products-admin.js:375]: @N-Molham do we need to update all these since values before release?

Comment (jconroy) [reply assets/js/admin/facebook-for-woocommerce-products-admin.js:375]: Checked with @wvega , we should update

**Comment by jconroy** [comment]
I've tested the changes described in #1683 and they appear to be working as expected

**Comment by jconroy** [comment]
Note to self - need to add #1752 to the changelog before release

**Comment by jconroy** [comment]
#1719 will be worth including

**Comment by wvega** [comment]
@jconroy I think is also worth reviewing and merging #1745 and https://github.com/facebookincubator/facebook-for-woocommerce/pull/1768. There is at least one merchant using a pre-release with those changes that would start seeing the error again if a new version is released without them.

**Comment by jconroy** [comment]
Updated to be release version 2.3.0

**Pull request closed without merging**